Homework_8_classes_objects_fields_methods/MyAmazingRPG/src/Character.java

```diff
@@ -0,0 +1,51 @@
+public class Character {
+
+    static int characterCount;
+
+    public Character() {
```

Comment:
One point here:
In Java, there's a convention of which members of the class should be written in which order (from top to the bottom)

1. Fields (Instance variables)
2. Constructors
3. Methods

As `public Character()` is a constructor method, this can be reordered a bit, so all the fields (both static and non-static) are grouped at the top of the Class definition, then Constructor, then the rest of the methods

I know I didn't tell it, but you might agree that it makes sense as it makes the class definition more readable :)